public/app/features/profile/ReactProfileWrapper.tsx

```diff
@@ -17,6 +18,7 @@ export const ReactProfileWrapper = () => (
             />
           )}
           <SharedPreferences resourceUri="user" />
+          {!states.loadTeams && <UserTeams loadTeams={api.loadTeams} teams={teams} />}
```

Comment:
I think this could be easier to understand if you passed states.loadTeams via isLoading prop to UserTeams and handled the loading state (show indictor?) in that component.

Reply:
Okay. I think this did it.